source/contact.rst

```diff
@@ -1,32 +1,25 @@
-.. _ref-contact:
+.. _contact:
 
 Contact
 =======
 
-The AppImage project documentation is brought to you by `the AppImage team <https://github.com/AppImage>`__. The source code is available `on GitHub <https://github.com/AppImage/docs.appimage.org>`__.
+The AppImage project documentation is brought to you by `the AppImage team <https://github.com/AppImage>`__ and other `contributors <https://github.com/AppImage/docs.appimage.org/graphs/contributors>`__. The source code is available `on GitHub <https://github.com/AppImage/docs.appimage.org>`__.
 
-This page outlines how you can contact the team behind AppImage, e.g., to get additional support or have questions answered.
+If you have further questions or need additional support, please feel free to contact the AppImage team:
 
 
-IRC
----
+GitHub discussions
+------------------
 
-If you have further questions, please feel free to contact the AppImage team. The easiest and fastest way is to join our `IRC channel #appimage <ircs://irc.libera.chat/appimage>`__ on `Libera.Chat <https://libera.chat>`__ (`webchat <https://web.libera.chat/#AppImage>`__).
+An easy way to contact us is to ask questions on the `AppImage GitHub discussions page <https://github.com/orgs/AppImage/discussions>`__. Questions are usually answered in one to two days, and you'll get notified as soon as someone answered you. Posts stay, so you can always keep track of the conversation. You can also browse through other posts to see if someone else had the same question or problem.
```

Comment:
Please do not include what seems to be an SLA, we might not be able to keep up with such a standard. If you stay more vague (e.g., "a few days"), I could live with it.

Reply:
I wrote usually as this was how I assessed it when looking at the Github discussions (questions were often even answered in just a few hours), so rather as an estimate and not as a guarantee. I can change it though if you want.